.github/workflows/release.yml

```
@@ -32,13 +32,41 @@ on:
           - minor
           - major
 jobs:
-  release:
+  rc:
```

pensavo se torna anche a te che potremmo semplificare se consideriamo che un rilascio latest viene fatto sempre e solo da master, dagli altri `branch` è invece possibile fare delle RC.
utilizzando github.ref https://docs.github.com/en/actions/learn-github-actions/contexts#github-context, potremmo evitare l'input di selezione per il rilascio `latest/rc` ed usare il github.ref per condizionare il job da eseguire.
branch master -> job `latest`
tutti gli altri branch ->job  `rc`
utilizzando `github.ref_name` potremmo evitare anche l'input del `branch_name` 

i job 
```suggestion
  jobs:
  latest:
    if: ${{ github.ref == 'refs/heads/master' }}
    .....
    .....

  rc:
    if: ${{ github.ref != 'refs/heads/master' }}
    ....
    .... 
        - name: Prepare package
        run: |
          git checkout ${{ github.ref_name }}
```